evap/staff/tests/test_views.py

```diff
@@ -465,6 +465,28 @@ def test_badge_for_external_responsibles(self):
         response = self.app.get(self.url, user="manager@institution.example.com")
         self.assertContains(response, 'External responsible')
 
+    def test_text_answer_review_urgent(self):
```

Comment:
Any ideas for better names that will not be more than 120 chars? Technically, this only tests that when the evluation is still in progress, the urgent-indicator is not shown, but when it is finished, it is...

Reply:
Actually, I’d expect the tests to look like this:

- In `test_models`, different evaluations would be created and tested on their `text_answer_review_is_urgent` property (may be also other properties, as those are more “scenario” tests)
- In `test_views`, you mock the `text_answer_review_is_urgent` property (which then could look like your test, but with different evaluation setup)

If you do not want to split the tests (which is really okay here, it is a rather small property), I would like a comment that the evaluation first is considered to be “not urgent” and after ending it to be “urgent”. Then the intend is clear and the name fits imo.